Feature onboarding

celerie opened the pull request:
onboarding screen

may need to check and draft if it doesn't work

celerie committed:
add test screens for notification and setting
adding introduction screen

celerie committed:
Changes by dyllon

celerie committed:
add test screens for notification and setting
adding introduction screen

celerie committed:
onboarding screen

celerie committed:
introduction page

RealDyllon committed:
Added intro screen

RealDyllon commented:
@celerie we need to update the text and graphics in the screen to make them more meaningful

celerie committed:
fixed onboarding

iAlanx3 committed:
Added "Input User's Home Address" to onboarding

RealDyllon committed:
Fix rebase issues

iAlanx3 committed:
Changed stateless into Stateful
Added textcontroller for passing HomeAddress data
To-Do:
1) Configure updateHomeAddress under setState upon tapping submit button
2) Configure router upon tapping submit button

iAlanx3 committed:
Added buttons for next and back

RealDyllon committed:
Add IntroScreen to router

RealDyllon commented:
@celerie my man let get this bread - let me know what the status of this is. cheers bud 🍻
